DiscordChatExporter.Core/Services/DataService.cs

```diff
@@ -63,6 +63,38 @@ public partial class DataService : IDisposable
             });
         }
 
+        public async Task<bool> GetAccessAsync(AuthToken token, string resource, string endpoint)
+        {
+            // Create request policy
```

Comment:
I would rewrite this to just try `GetApiResponseAsync` and catch `HttpErrorStatusCodeException` with status code 403.
Also, rename the method to `CheckAccessAsync` maybe.

Reply:
Or just not use it and have a try/catch in the command `ExecuteAsync` which swallows 403 exceptions.